specs/pvs/corla.pvs

```diff
@@ -394,24 +409,37 @@ END election
 
 %+BEGIN ballots
 
-%%% \section{Ballots}
+%%% \subsection{Ballots}
 
 %%% The second basic concept of elections that we need to specify is
-%%% the ballot. Ballots are the means by which voters indicate their
-%%% preferences for candidates in contests and choices in ballot
-%%% questions.
+%%% the ballot. A ballot is a (possibly flawed) representation of the choices
+%%% made by a voter in all contests in the election.
+%%% A ballot may be paper, or it may be electronic -- for example in Colorado
+%%% UOCAVA voters can return their voted ballots by email. In Colorado these electronic ballots 
+%%% are printed out, producing paper ballots. In some situations, say, if a
+%%% voter has marked a paper ballot with ink of a color that the scanners cannot read,
+%%% Election Board workers will create a duplicate ballot from the voter's original.
+% @todo sfsinger find sources for these, other than "Dwight Shellman said so".
+%%%
+% @review kiniry dmz I think of contests_with_candidates and ballot_questions
+% as subsets of "contests". A ballot question is a contest between "yes" and "no".
 %%%
-%%% Ballots are stored by one of several possible means. Some ballot
+%%% There are a variety of ways to store ballots. Some ballot
 %%% storage containers are secure, others are not. Some facilitate an
 %%% easy means by which to find a particular ballots, others do not.
 
 % @relation box inherit storage_container
 % @relation bin inherit storage_container
 % @relation ballot_box inherit storage_container
```

Comment:
I was distinguishing them because (a) I had witnessed different storage mechanisms in the field, (b) I understand there is not a uniform mechanism in CO, and (c) some mechanisms are secure and provide a chain-of-custody evidence and others are not.  This latter point matters wrt the meaning of `secure` and `verified`.